### PR TITLE
fix: scribbled style shape rendering

### PR DESCRIPTION
**Pull request by doouding**
Fixes [BS-647](https://linear.app/affine-design/issue/BS-647/%E8%80%81%E6%95%B0%E6%8D%AE%E7%9A%84-shape-%E5%9C%A8-beta-canary-%E4%B8%8A-scribbled-%E7%9A%84%E5%BD%A2%E7%8A%B6-%E4%B8%8D%E7%94%9F%E6%95%88)

### CODE REVIEW

**Comment by doouding** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7420?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#7420** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7420?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#7415** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7415?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @doouding and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/blocks/src/surface-block/surface-model.ts [hunk 1]
```diff
@@ -136,6 +136,21 @@ const migration = {
       const wrapper = new Boxed(yMap);
       data.elements = wrapper;
     }
+
+    const childrenMap = data.elements.getValue() as Y.Map<Y.Map<unknown>>;
```
Comment (doodlewind) [inline packages/blocks/src/surface-block/surface-model.ts:140]: I think we should avoid direct operating on YTypes.

Comment (doouding) [reply packages/blocks/src/surface-block/surface-model.ts:140]: > I think we should avoid direct operating on YTypes.

This happens during migration. Not sure if there's better way. cc @Saul-Mirone

**Comment by Saul-Mirone** [comment]
Maybe add e2e test?

**Comment by doouding** [comment]
> Maybe add e2e or integration test?

It's hard to assert if element is drawn in canvas or not. This can be done once the canvas-related test suite is configured.

**Review by Saul-Mirone — APPROVED** [review]

**Pull request merged**